melos.yaml

```diff
@@ -20,6 +20,7 @@ command:
         dart run scripts/generate_vertexai_version.dart && \
         dart run scripts/generate_dataconnect_version.dart && \
         dart run scripts/generate_versions_spm.dart && \
+        dart run scripts/generate_versions_gradle.dart && \
```

Comment:
lets remove it from here, this is for versioning. should just be a plain script in melos we can run when we want to bump them